shared/rakefiles/xk_util.rake

```diff
@@ -127,6 +127,58 @@ task :display_cluster_state => [:configure, :configure_secrets, :set_secrets] do
   end
 end
 
+task :display_universal_image_info => [:configure] do
+  sh "#{@exekube_cmd} sh -c ' \
+    UNIVERSAL_CI_URL=\"https://ci.gpii.net\";
+    UNIVERSAL_REPO=\"https://github.com/gpii/universal\";
+    RELEASE_JOB_URL=\"$UNIVERSAL_CI_URL/job/docker-gpii-universal-master-release\";
+    UPSTREAM_JOB_URL=\"$UNIVERSAL_CI_URL/job/docker-gpii-universal-master\";
+    LOOKUP_BUILDS=\"20\";
+
+    PREFERENCES_IMAGE_SHA=$(kubectl -n gpii get deployment preferences -o json 2> /dev/null | jq -r \".spec.template.spec.containers[0].image\" | grep -o \"sha256:.*\");
+    if [ \"$?\" != \"0\" ]; then
+      echo
+      echo \"Unable to retrieve data from K8s cluster!\";
+      echo \"Try running \\\`rake display_cluster_state\\\` for debug info.\";
+      echo
+      exit 1;
+    fi
+
+    echo
+    echo \"Preferences image SHA:\";
+    echo \"$PREFERENCES_IMAGE_SHA\";
+    RELEASE_BUILD=$(curl $RELEASE_JOB_URL/lastBuild/api/json 2> /dev/null | jq -r \".id\");
+    RELEASE_BUILD_LIMIT=$(($RELEASE_BUILD-$LOOKUP_BUILDS));
```

Comment:
I would add spaces around `-` to make it clear this is arithmetic subtraction, not joining strings together.

Reply:
Yeah, `shellcheck` suggests the same.